h/static/scripts/annotator/guest.coffee

```diff
@@ -1,15 +1,7 @@
-$ = require('jquery')
 Annotator = require('annotator')
-Channel = require('jschannel')
+$ = Annotator.$
 
-require('diff-match-patch')
```

Comment:
I'm not sure I see the logic of moving these into `hypothesis.js`. Doesn't it make most sense to keep these requirements as close to the code that actually uses them as possible?